source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,93 @@
+---
+title: "Core Concepts"
+description: "How Home Assistant works."
+---
+
+Home Assistant, like all home automation software, allows you to control smart devices in your home. This page will explain the parts of Home Assistant and how they relate.
+
+## Devices
+
+In Home Assistant, a **device** is a physical object in your connected home. This would be something like a smart lightbulb or a Chromecast.
+
+Home Assistant has a **discovery** feature, which scans for new devices constantly and adds them to your Home Assistant automatically.
+
+**Device trackers** are devices that track the position of something or someone, like a person or a cellphone.
+
+## Entities
+
+Because many devices do more than one thing, things in Home Assistant are controlled using **entities**. An entity is a single function or property of a device, like a single sensor, light, or switch.
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
+
+To get entities to appear in Home Assistant, you must define them in your [`configuration.yaml` file](/docs/configuration) in this format: `entity_id: domain.object_id`. The **domain** is a built-in category of entities that you manage in a similar way. You can get the object ID by ________?
```

Comment:
Could I have some help with this sentence?

Reply:
line 20 is great.  Line 22 seems like it's out of place.  That format will only be used in specific spots in configuration.yaml.

Reply:
Thanks @Petro31 , definitely struggling with line 22. How's something like this?

```
To get entities to appear in Home Assistant, you must define them in your [`configuration.yaml` file].

The **domain** is a built-in category of entities that you manage in a similar way. For example, all lights are part of the "light" domain.
```